CampaignConfig/mdc2020_trkana.cfg

```diff
@@ -0,0 +1,33 @@
+[global]
+includes = /mu2e/app/users/sophie/ProductionManager/Production/CampaignConfig/mdc2020_prolog.cfg
+release = MDC2020
+desc = MDC2020s
+release_v_i = r
+release_v_o = s
+db_version = v1_0
+db_purpose = perfect
+primary_name = IPAMuminusFlateMinus
```

Comment:
The primary name is more specific than the name of this file would imply

Reply:
Yes, the primary can be edited to whatever the used wants, this was just an example